mlrun/config.py

```diff
@@ -114,6 +114,7 @@
     "v3io_api": "http://v3io-webapi:8081",
     "redis": {
         "url": "",
+        "type": "",
```

Comment:
add comment of supported redis types which points to the class you created

Reply:
also is this param required? or can we infer it by some way?

Reply:
> add comment of supported redis types which points to the class you created

added comment.

> also is this param required? or can we infer it by some way?

in theory we can try one type, and if does not work (we get an error) try the other type. i don't think we want this.